fedimintd/templates/params.html

```diff
@@ -54,7 +54,7 @@ <h5>Network connectivity</h5>
         class="form-control"
         id="p2p_url"
         name="p2p_url"
-        placeholder="wss://p2p.your-website.com"
```

Comment:
Can we keep these placeholders? If `""` gets passed in the placeholder should still display.

Reply:
Sure, I've added the placeholders back in, which will be displayed if the corresponding value is empty.

Regarding the values, I've included two different approaches. One checks for an empty string and the other checks for `Some`/`None`. You can let me know which way you prefer and I can make them all consistent.

Reply:
I don't think you even need the if-else. Just do it like this:

```
        value={{s}}
        placeholder="..."
```

Reply:
Ah yes, you're right. When I initially tried that the value was showing up as `/`, but I think it's because I wasn't enclosing it in quotes. I've now made this change using quotes around the value.

I've also removed the use of `Option` and just default to an empty string again.